src/gba/ppu.nim

```diff
@@ -192,7 +192,7 @@ proc composite(pram: array[0x400, uint8], scanline: ptr Scanline) =
 
 proc getLine(buffer: var FrameBuffer, row: SomeInteger): ptr Scanline =
   result = cast[ptr array[buffer.width, buffer.T]](addr buffer[buffer.width * row])
-  for pixel in result[].mitems: pixel = 0
+  reset(result[])
```

Comment:
Oh nice! Didn't know about reset. I can definitely do this in `resampler.nim` as well where I currently do `resampler.data.fill(cast[T](0))`